starknet_devnet/blueprints/base.py

```diff
@@ -44,7 +46,17 @@ def extract_positive(request_json, prop_name: str):
     return value
 
 
-def extract_hex_string(request_json, prop_name: str) -> int:
+def to_int(value) -> int:
+    """Convert to int value"""
+    return int(value, 16)
+
+
+def to_int_array(value) -> List[int]:
+    """Convert to List of ints"""
+    return [int(numeric, 16) for numeric in value]
+
+
+def extract_hex_string(request_json, prop_name: str, fun=to_int) -> int:
```

Comment:
This method no longer does what its name implies (and the error message used in case of `ValueError` or `TypeError` is not true if parsing a list).

`fun` is not a good name

Reply:
https://github.com/Shard-Labs/starknet-devnet/pull/365/commits/16f78dee7af6445a6db974f3a7475ae62e2d7ffb